blogs/2025/07/16/copilot-coding-agent.md

```diff
@@ -0,0 +1,112 @@
+
+![MCP configuration shown for Copilot Coding Agent on github.com](coding-agent-mcp-config.png)
+
+But it gets even better - because you can do all of this right from VS Code.
+
+### Start sessions from Copilot Chat
+
+We find ourselves wanting to do more and more right from the [chat panel](https://code.visualstudio.com/docs/copilot/chat/copilot-chat) in VS Code. While we love working in GitHub, many of us find that VS Code is our home base. Bringing Coding Agent into chat means you're not locked into the issue workflow. Now, you can - at any point in a chat, no matter how long - delegate the task you're working on directly to the Coding Agent. All of the context that's part of your current chat session will be handed off to the Coding Agent.
+
+The Coding Agent then opens a PR and goes directly to work - skipping the issue step entirely. It distills down the context into a detailed PR description and you'll see that Copilot Coding Agent makes use of "to-do lists" so that you can see what it's doing and where it is in the process at a glance.
+
+<video src="delegate-to-coding-agent.mp4" title="Delegate to Coding Agent video" controls poster="delegate-to-coding-agent.png"></video>
+
+## What’s next?
+
+The Copilot Coding Agent integration in Visual Studio Code is currently in preview, but you can use it all today. As a reminder, make sure you have the GitHub Pull Requests extension installed and add the following setting...
+
+```json
+"githubPullRequests.codingAgent.uiIntegration": true,
+```
+
+We are very excited to further deepen this integration with Coding Agent. And since we recently made the move [to fully open source the AI features in VS Code](https://code.visualstudio.com/blogs/2025/05/19/openSourceAIEditor), you can track our progress in our [Coding Agent iteration plan](https://github.com/microsoft/vscode/issues/255483). We're working on:
+- PR performance and rendering
+- Integrated chat view for Coding Agent sessions
+- Bringing the Copilot Agents command center into VS Code
+- Sharing custom instructions between the Coding Agent and VS Code
+- Adding more documentation
+
+We'd love your thoughts on the experience, so please provide any feedback as [issues in the VS Code repo](https://github.com/microsoft/vscode/issues)!
+
+## Go forth and 100x yourself
+
+We can’t wait to see what you and your new agent friends build together! Keep an eye on the [release notes](/release-notes/) and the [Copilot documentation](https://docs.github.com/en/copilot/concepts/about-copilot-coding-agent) for new developments here.
+
+In the meantime, start making a list of all the things you’d rather not do. Because you’re about to have a new favorite squad of teammates.
```

Comment:
This is good - we can also go with

```suggestion
In the meantime, start making a list of all the things you’d rather not do. Because you’re about to have a few new favorite teammates.
```